config/environments/production-legacy.rb

```diff
@@ -0,0 +1,95 @@
+Rails.application.configure do
```

Comment:
It occurs to me we could, rather than saving this file here (which makes me nervous because any file dropped in here becomes a legitimate environment) we could put a comment in production.rb with a permalink to the version of production.rb copied here? 

I'd prefer it pretty strongly, having an outdated environment that could be applied on app startup, however unlikely, feels a little risky.

Reply:
Good call, I've changed these to text files instead